### PR TITLE
Change KEA3 stacked bar chart to display sum of scaled and normalized ranks

### PR DESCRIPTION
**Pull request by serena-zhang**

### CODE REVIEW

File: appyters/KEA3_Appyter/KEA3-Appyter.ipynb [hunk 1]
```diff
@@ -169,95 +169,110 @@
     "    all_prePPI = []\n",
     "    all_PTMsigDB = []\n",
     "    \n",
-    "    for kinase in range(0, 10):\n",
+    "    # Length of all the libraries (for calculation of )\n",
+    "    max_ranks = {}\n",
+    "    for library in range(2,13):\n",
+    "        lib_name = (list(results.keys())[library])\n",
```
Comment (u8sand) [inline appyters/KEA3_Appyter/KEA3-Appyter.ipynb:175]: This looks really strange and is hard to follow. Typically if you're using integer indexes instead of string ones you're doing something strange.

here you should just do:
```python
for lib_name in results.keys():
  # ...
```

File: appyters/KEA3_Appyter/KEA3-Appyter.ipynb [hunk 2]
```diff
@@ -169,95 +169,110 @@
...
+    "        max_ranks.update({lib_name: len(list(results.values())[library])})\n",
```
Comment (u8sand) [inline appyters/KEA3_Appyter/KEA3-Appyter.ipynb:176]: Same thing here, it would just be: `max_ranks[libname] = len(results[lib_name])`

**Comment by serena-zhang** [comment]
I have made your suggested changes. Thank you so much!
